spacy/ml/models/entity_linker.py

```diff
@@ -1,11 +1,17 @@
 from pathlib import Path
-from typing import Optional, Callable, Iterable, List, Tuple
+from typing import Optional, Callable, Iterable, List, Tuple, Iterator, Union, Type
 from thinc.types import Floats2d
-from thinc.api import chain, clone, list2ragged, reduce_mean, residual
-from thinc.api import Model, Maxout, Linear, noop, tuplify, Ragged
+from thinc.api import chain, list2ragged, reduce_mean, residual
+from thinc.api import Model, Maxout, Linear, tuplify, Ragged
 
 from ...util import registry
-from ...kb import KnowledgeBase, Candidate, get_candidates
+from ...kb_base import (
+    BaseKnowledgeBase,
+    Candidate,
+    get_candidates,
+    get_candidates_batch,
+)
+from ...kb import KnowledgeBase
```

Comment:
When a line gets too long like this, we repeat the `from ... import` bit so it becomes 2 lines. You already have the second line, so you can shift one or more two imports onto that one and get rid of the `()` again :-)